app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -138,32 +133,9 @@ class HomeFragmentPresenter @Inject constructor(
     // If there's an error loading the data, assume the default.
     return Transformations.map(topicListSummaryResultLiveData) { it.getOrDefault(TopicList.getDefaultInstance()) }
   }
-
-  private fun subscribeToUserAppHistory() {
-    getUserAppHistory().observe(fragment, Observer<UserAppHistory> { result ->
-      userAppHistoryViewModel = UserAppHistoryViewModel()
-      userAppHistoryViewModel.setAlreadyAppOpened(result.alreadyOpenedApp)
-      setProfileName()
-      itemList[0] = userAppHistoryViewModel
-      topicListAdapter.notifyItemChanged(0)
-    })
-  }
-
-  private fun getUserAppHistory(): LiveData<UserAppHistory> {
-    // If there's an error loading the data, assume the default.
-    return Transformations.map(userAppHistoryController.getUserAppHistory(), ::processUserAppHistoryResult)
-  }
-
-  private fun processUserAppHistoryResult(appHistoryResult: AsyncResult<UserAppHistory>): UserAppHistory {
-    if (appHistoryResult.isFailure()) {
-      logger.e("HomeFragment", "Failed to retrieve user app history" + appHistoryResult.getErrorOrNull())
-    }
-    return appHistoryResult.getOrDefault(UserAppHistory.getDefaultInstance())
-  }
-
   private fun setProfileName() {
```

Comment:
Add one blank line above this. There should be one line gap between two functions.

Reply:
Done :)